tinyssh_install

```diff
@@ -72,6 +79,12 @@ build ()
   umask 0022
 
   copy_openssh_keys || generate_keys
+
+  if ! verify_keys ; then
+    error_no_keys
+    return 1
+  fi
+
   display_fingerprints
```

Comment:
At first I wanted to add the check inside display_fingerprints but I figured it made more sense to separate concerns.